src/nile/core/compile.py

```diff
@@ -11,12 +11,14 @@
 )
 
 
-def compile(contracts, directory=None, account_contract=False):
+def compile(contracts, disable_hint_validation, directory=None, account_contract=False):
```

Comment:
```suggestion
def compile(contracts, directory=None, account_contract=False, disable_hint_validation=False):
```
I suggest moving this option to the end of the argument list since I expect the other ones to be used more often. Also, having it first forces users to pass a value even if they don't care about the option.

Reply:
Sure lets move it to the end though as it is a flag if you don't add it it'll automatically be set to `False`

Reply:
Not necessarily, Nile can be used programmatically not just through CLI

Reply:
see https://github.com/OpenZeppelin/nile/#run